tinymce-language-selector/plugin.js

```diff
@@ -131,6 +129,17 @@ tinymce.PluginManager.add('language', function(editor) {
         }
         updateButtonText(selectedLang);
       };
+      editor.addShortcut('Meta+L', 'Switch to default language', () => {
+        const selectedNode = editor.selection.getNode();
+        const currentLang = selectedNode.lang ? selectedNode.lang : BROWSER_DEFAULT;
+        if (currentLang !== BROWSER_DEFAULT) {
+          editor.undoManager.transact(() => {
+            replaceText(BROWSER_DEFAULT);
+            buttonApi.setActive(false);
+            updateCurrentLanguage();
```

Comment:
Above you didn't call this and now you are. Why the change?

Reply:
I thought that the `keyup` hook would catch this case, but if you Cmd-L out of a language, it doesn't update the button text until you keep typing. (This should've been in the previous PR but I missed it.)